companies/phoenixcontact.json

```
@@ -0,0 +1,21 @@
+{
+    "slug": "phoenixcontact",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Phoenix Contact",
+    "address": "Flachsmarktstraße 8\n32825 Blomberg\nDeutschland",
+    "phone": "+49 5235 300",
+    "fax": "+49 5235 341200",
+    "email": "corporatewebsite@phoenixcontact.com",
```

```suggestion
    "email": "datenschutz@phoenixcontact.com",
```

from https://dam-mdc.phoenixcontact.com/asset/156443151564/8a6e5eb157289925b365b3247a31c5f0/Phoenix-Contact-List-of-DPO.pdf

Could you please add that as a source as well and set the suggested-transport-medium :D